lib/orbit-common/cache.js

```diff
@@ -171,6 +171,25 @@ var Cache = Class.extend({
     var value = normalizedOperation.value;
     var currentValue = this.retrieve(path);
     var _this = this;
+    var dependentOperations = [];
+    var pushOps = function(ops) {
+      if (ops) {
+        if (ops.forEach) {
+          ops.forEach(function(op) {
+            if (op) dependentOperations.push(op);
+          });
+        } else {
+          dependentOperations.push(op);
+        }
+      }
+      return dependentOperations;
+    };
+
+    var popOps = function() {
```

Comment:
Should this method clear reset dependentOperations to `[]` after the forEach? I see two calls to `popOps`. Does the second expect to rerun any operations the first call ran? If it does expect to rerun, maybe this method should be renamed since the current implementation is kind of occluded from the meaning I get from the name `popOps`.